be/src/vec/data_types/number_traits.h

```diff
@@ -220,7 +221,6 @@ template <typename T>
 /// Returns the maximum ascii string length for this type.
 /// e.g. the max/min int8_t has 3 characters.
 int max_ascii_len() {
-    LOG(FATAL) << "Not implemented.";
```

Comment:
why not add dcheck or exception?

Reply:
On be/src/vec/functions/function_cast.h line 280, I use max_ascii_len to judge multiply may overflow